docs/loadflow/loadflow.md

```diff
@@ -149,3 +149,66 @@ Note that the vector $b$ of right-hand sides is linearly computed from the given
 
 To solve this system, we follow the classic approach of the LU matrices decomposition $J = LU$.
 Hence, by solving the system using LU decomposition, you can compute the voltage angles by giving as data the injections and the phase-shifting angles.
+
+## Area Interchange Control
+
+The area interchange control can be activated in the parameters and is performed by an outer loop.
+
+The outer loop performs an active power distribution in each one of the selected areas (for areas selection see `areaInterchangeControlAreaType`),
+in order to have all areas' active power interchanges matching their target interchanges.
+It can handle networks where part (or even all) of the buses are not in an area. For networks that have no areas at all, the behaviour will be the same as with the distributed slack outer loop.
+
+It is preformed in an iterative way. After each iteration, a load flow computation is run and the next iteration will apply its changes based on this new calculated network state.
```

Comment:
performed ? (instead of preformed)

Reply:
thanks ! fixed in e5d9d8d1e3e29980fbe2d62de80be0cf53de1340